src/Filesystem/Provider/FlySystem/FlySystemLocalFilesystemFactory.php

```diff
@@ -50,7 +51,8 @@ public function getInstance(LocalConfig $config): \ILIAS\Filesystem\FilesystemFa
             $config->getFileAccessPublic(),
             $config->getFileAccessPrivate(),
             $config->getDirectoryAccessPublic(),
-            $config->getDirectoryAccessPrivate()
+            $config->getDirectoryAccessPrivate(),
+            Visibility::PUBLIC_ACCESS
```

Comment:
I am looking at this in code.
It appears League\Flysystem\Visibility\Visibility has "PUBLIC", not "PUBLIC_ACCESS".

Reply:
Just see....  ILIAS\Filesystem\Visibility\Visibility has PUBLIC_ACCESS, so nevermind. My bad.